### PR TITLE
fix(plugin-meetings): fix meetingInfo 404 error

### PR DESCRIPTION
**Pull request by arun3528**
As some of the locus URL are federated so we should not use the locus service from catalog for fetching the locus info

### CODE REVIEW

File: packages/node_modules/@webex/plugin-meetings/src/meeting-info/util.js [hunk 1]
```diff
@@ -222,7 +223,7 @@ MeetingInfoUtil.getResourceUrl = (type, value) => {
       method = HTTP_VERBS.PUT;
       break;
     case _LOCUS_ID_:
-      resource = `/${LOCI}/${value}/${MEETINGINFO}`;
+      uri = `${value}/${MEETINGINFO}`;
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-meetings/src/meeting-info/util.js]: Does this mean that a locus id request has a full locus url in it?

Comment (chrisadubois) [reply packages/node_modules/@webex/plugin-meetings/src/meeting-info/util.js]: yes

File: packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting-info/request.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+import Mercury from '@webex/internal-plugin-mercury';
+import Meetings from '@webex/plugin-meetings/src/meetings';
+import {skipInBrowser} from '@webex/test-helper-mocha';
+import {
+  _LOCUS_ID_
+} from '@webex/plugin-meetings/src/constants';
+
+import MeetingInfo from '../../../../src/meeting-info/index';
+
+skipInBrowser(describe)('plugin-meetings', () => {
```
[the remark this answers is not in the document]
Comment (chrisadubois) [reply packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting-info/request.js]: this was a copy/paste error im guessing pushing a commit

File: packages/node_modules/@webex/plugin-meetings/src/meeting-info/request.js [hunk 3]
```diff
@@ -24,9 +24,8 @@ export default class MeetingInfoRequest extends StatelessWebexPlugin {
     if (!options || !options.type || !options.destination) {
       throw new ParameterError('MeetingInfo should be fetched with a type and destination specified, see list of valid types and their corresponding values in constants');
     }
-    const format = MeetingInfoUtil.getMeetingInfoAPIFormat(options);
-    const resourceUrl = MeetingInfoUtil.getResourceUrl(format.type, format.value);
-    const requestParams = MeetingInfoUtil.getRequestParams(resourceUrl, format.type, format.value, API.LOCUS);
+    const resourceUrl = MeetingInfoUtil.getResourceUrl(options.type, options.destination);
+    const requestParams = MeetingInfoUtil.getRequestParams(resourceUrl, options.type, options.destination, API.LOCUS);
```
Comment (InteractiveTimmy) [inline packages/node_modules/@webex/plugin-meetings/src/meeting-info/request.js]: For readability, could this be refactored to

```js
const requestParams = MeetingInfoUtil.getRequestParams(
  resourceUrl,
  options.type,
  options.destination,
  API.LOCUS
);
```

Comment (arun3528) [reply packages/node_modules/@webex/plugin-meetings/src/meeting-info/request.js]: will do it in the next PR . I don't want to re run the test

**Pull request merged**